src/page.tsx

```diff
@@ -0,0 +1,20 @@
+import { ComponentChildren, VNode } from "preact";
+import Header from "./header";
+
+interface PageProps {
+  path: string;
+  children?: ComponentChildren;
+  title: string;
+}
+
+function Page({ path, children, title }: PageProps): VNode {
+  return (
+    <>
+      <Header path={path} />
+      <h1>{title}</h1>
```

Comment:
This addresses the comment here: https://github.com/18F/identity-reporting/pull/23#discussion_r710091303

Even better than rendering it per-route, imo.